### PR TITLE
Set post defaults for ease of generation of posts in a dev environment

### PR DESCRIPTION
**Pull request by Throne3d**
- Set last_user in a before_create callback (instead of the controller)
- Set default content to be ''

### CODE REVIEW

File: db/migrate/20161229204445_set_default_content_for_posts.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class SetDefaultContentForPosts < ActiveRecord::Migration
+  def up
+    change_column_default :posts, :content, ''
```
Comment (Marri) [inline db/migrate/20161229204445_set_default_content_for_posts.rb]: If this is literally only useful when manually creating posts in the console, I'd rather skip the database migration, they're kind of a pain.

Comment (Throne3d) [inline db/migrate/20161229204445_set_default_content_for_posts.rb]: I'm not too strongly opinionated about *when* it gets merged, so… we could just wait until we plan on doing something else in a migration?

Comment (Marri) [inline db/migrate/20161229204445_set_default_content_for_posts.rb]: This shouldn't need a migration at all, though? Do we just need a .to_s somewhere, or something, this isn't something that ever happens in the actual correct user flow or even really in tests.

Comment (Throne3d) [inline db/migrate/20161229204445_set_default_content_for_posts.rb]: Hmm, good point – I'll look into it tomorrow.

**Comment by Throne3d** [comment]
Now uses a `.to_s` call like suggested.

**Pull request merged**